15/umbraco-cms/tutorials/creating-and-distributing-a-package.md

```diff
@@ -233,13 +233,13 @@ Now your package is on Our, and if the "Go live" button is clicked it is visible
 
 The next step is to make it a bit simpler to deploy updates to the package. It is perfectly fine to log in here and upload a new version each time. The next steps will show an easier way though.
 
-## Pushing your package to Github
+## Pushing your package to GitHub
 
 If you are creating a package in order to share it with others it is a great idea to also share the source code. It is the open source way.
 
-To share it, and make it easier to manage and deploy updates we will set up a Github repository for the package. This tutorial assumes you know what Github is, and that you have an account.
+To share it, and make it easier to manage and deploy updates we will set up a GitHub repository for the package. This tutorial assumes you know what GitHub is, and that you have an account.
 
-Create a fresh repo, with no readme, gitignore or license - do not choose a repository template (set to 'No Template'). On the second screen it will give you a command to push an existing repository to the new Github repo, should look like this but with your own user in the link:
+Create a fresh repo, with no readme, gitignore or license - do not choose a repository template (set to 'No Template'). On the second screen it will give you a command to push an existing repository to the new GitHub repo, should look like this but with your own user in the link:
```

Comment:
```suggestion
Create a fresh repo, with no readme, gitignore or license - do not choose a repository template (set to 'No Template'). The second screen will provide a command to push an existing repository to your new GitHub repo. It will look like this but with your own username in the link:
```